service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/UploadAdditionalApplicationsSubmittedControllerTest.java

```diff
@@ -258,35 +255,22 @@ void submittedEventShouldNotifyHmctsAdminAndRepresentativesWhenCtscToggleIsDisab
             anyMap(),
             eq(NOTIFICATION_REFERENCE)));
 
-        checkUntil(() -> verify(sendLetterApi, times(2))
+        checkUntil(() -> verify(sendLetterApi, times(1))
```

Comment:
I believe times(1) is the default. Unless you want to make it clear, you could remove it.